backend/pdf.py

```
@@ -1,45 +1,90 @@
+            return None
+
+        # Берем только первую страницу
+        page = doc[0]
+        # dpi=72 делает картинку маленькой и быстрой для загрузки в списке
+        pix = page.get_pixmap(dpi=72)
+        pix.save(output_path)
+        doc.close()
+
+        logger.info(f"Thumbnail successfully saved to {output_path}")
+        return output_path
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)